components/director/internal/domain/labeldef/resolver.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+var defaultTemplateName = "Side-by-side extensibility with Kyma"
```

Comment:
Could be moved to `internal/model/formation.go` in a similar way to the ScenarioKey in `scenarios.go`, but it's ok here for now.